src/pages/Chart/Chart.tsx

```diff
@@ -2806,6 +2806,14 @@ export default function Chart(props: propsIF) {
                 setMainCanvasBoundingClientRect(canvas.getBoundingClientRect());
 
                 const height = result[0].contentRect.height;
+                // height given here doesn't account for constant 113 px of header elements in chart
+                // add it here so when we set the height of the whole chart via it's default
+                // it will be correct
+                const heightToSave = height + 113;
```

Comment:
113 was found more by inspection rather than knowing exactly where that height comes from. 

If it's a useful height calculated and stored elsewhere - would love to use that variable. 

Also should check across a few screen setups + mobile to ensure 113 px holds across the board.